.github/ISSUE_TEMPLATE/docs.yml

```diff
@@ -0,0 +1,11 @@
+name: Docs
```

Comment:
Do you want me to update the `contact_links` section?

https://github.com/renovatebot/renovate/blob/f54ff20170a96e8c898d891c292fab0a449395aa/.github/ISSUE_TEMPLATE/config.yml#L1-L5

It still says that docs changes should go to discussions, but if we also show a form in the issue select screen, it will get confusing. 😄

Reply:
Ideally I'd like to avoid issues for docs in the majority of cases where it's almost as fast to just submit a PR as it is to describe what's needed to change in an issue. So for most user-submitted cases I'd like their docs query to go to Discussions.

Would it work if for docs issues you create - for ones you don't have time to submit a PR directly - you could start with the refactor template? That way we could reduce the confusion for end users

Reply:
Checking if I understand correctly. I think this is what you want to end up with?

- Small docs change by @HonkingGoose -> Open PR rightaway, don't bother with discussion or issue
- Large docs change by @HonkingGoose -> use refactor issue form -> change label from `type:refactor` to `type:docs` myself (can do this with `triage` rights 😄 )
- User submitted documentation proposals -> goes into discussion.
- Remove `docs.yml` from PR, keep `config.yml` as it is right now with the redirect to the discussion for docs changes by the public

Reply:
Yep, do you think that will work ok?

Reply:
Let's try it out and see! 😄 

The `docs.yml` form is really similar to the `refactor.yml` form, so if we do find out later that we want a `docs.yml` form, it'll be easy to copy/paste and adjust things. 😄